doc/sphinx/addendum/type-classes.rst

```diff
@@ -350,6 +354,13 @@ Summary of the commands
    This command supports the :attr:`local`, :attr:`global` and :attr:`export`
    locality attributes.
```

Comment:
Maybe add some text to specify that inside sections the default is `local` and outside it is `global`.

Reply:
Gah. I wrote that before, @jfehrle complained this should just be a pointer and now I should write that again? Could you two first agree before writing this documentation snippet?

Reply:
My comment was probably in a different PR.

Reply:
If the default varies across commands, then maybe it should be mentioned.  It could be "Unlike most commands, the default is `local`" if that's correct.  If it behaves like most commands, I would not mention it.

Reply:
There is no known default across commands at this point.

Reply:
@ppedrot If you do not want to apply this suggestion, let me know and I'll merge as is.

Reply:
I'd rather do this in another PR, it's not quite related to this one. If anything, it should have gone into #14148 instead.

Reply:
@ppedrot I think you're referring to https://github.com/coq/coq/pull/14148/files/745d08eaa4d1d71aa51c300890653841c1af9390#diff-755d7b2b617de85c8a360c9be9fe816c6c86e183aaa5ce94b3ec8721ad472a32, where @jfehrle requested that the explanation of what each attribute does not be replicated.  Here I believe @Zimmi48 is requesting that the default be specified for each command (but not that the explanation of the meaning of each attribute be repeated).